### PR TITLE
Layers and IcoFont

### PR DESCRIPTION
**Pull request by sjmiles**
First half of sun-day work, then second half.

### CODE REVIEW

File: pkg/Library/CoreDesigner/DesignApp.js [hunk 1]
```diff
@@ -168,7 +168,7 @@ export const rebuildObject = async (layer, id) => {
 };
 
 const connectionsByObjectId = (graph, id) => {
-  const prefix = `${id}$`;
+  const prefix = `${id}\$`;
```
Comment (mariakleiner) [inline pkg/Library/CoreDesigner/DesignApp.js:171]: i'm just curious - does the escaping really matter? it's not a regex, just a string...

File: pkg/Library/OpenAI/OpenAINodeTypes.js [hunk 2]
```diff
@@ -35,7 +35,8 @@ export const OpenAINodeTypes = {
       OpenAIText$context: 'MultilineText',
       OpenAIText$prompt: 'MultilineText',
       OpenAIText$result: 'String',
-      OpenAIText$restart: 'Boolean',
+      OpenAIText$enabled: 'Boolean',
+      OpenAIText$restart: 'Nonce',
```
Comment (mariakleiner) [inline pkg/Library/OpenAI/OpenAINodeTypes.js:39]: thank you so much! 🙏

File: pkg/Library/UX/UXNodeTypes.js [hunk 3]
```diff
@@ -31,5 +31,10 @@ export const UXNodeTypes = {
     category,
     description: 'Collects UX action definitions for buttons and controls',
     type: `$library/UX/Nodes/UXActionSetNode`
+  },
+  UXSelectionList: {
+    category,
+    description: 'Render a list, allow user to select items',
```
Comment (mariakleiner) [inline pkg/Library/UX/UXNodeTypes.js:37]: nit - i think all (most) rest of the descriptions are in 3rd person "Renders a list..."; or if you prefer it this way, i can update the rest of them to match

**Review by mariakleiner — APPROVED** [review]
🔥

File: pkg/Library/CoreDesigner/DesignApp.js (hunk 1 above)
Comment (sjmiles) [reply pkg/Library/CoreDesigner/DesignApp.js:171]: The syntax highlighter in Chrome DevTools was confused by the non-escaped $ ...

File: pkg/Library/UX/UXNodeTypes.js (hunk 3 above)
Comment (sjmiles) [reply pkg/Library/UX/UXNodeTypes.js:37]: No preference, my mistake.

File: pkg/Library/Chrome/extension-v2/assets/logo_32x32.png
[the remark this answers is not in the document]
Comment (sjmiles) [reply pkg/Library/Chrome/extension-v2/assets/logo_32x32.png:1]: updated

File: pkg/Library/UX/UXNodeTypes.js (hunk 3 above)
Comment (sjmiles) [reply pkg/Library/UX/UXNodeTypes.js]: changed it to `Renders`

**Review by mariakleiner — APPROVED** [review]
I think we are good to land this now, thanks for waiting

**Pull request merged**